single-line-diagram-core/src/test/java/com/powsybl/sld/TestShiftFeedersPosition.java

```diff
@@ -162,15 +86,9 @@ public void test() {
         // build substation graph
         SubstationGraph g = graphBuilder.buildSubstationGraph(substation.getId(), false);
 
-        // write SVG and compare to reference (horizontal layout and defaut style provider)
+        // write Json and compare to reference (with horizontal substation layout)
         new HorizontalSubstationLayoutFactory().create(g, new PositionVoltageLevelLayoutFactory()).run(layoutParameters);
-        compareSvg(g, layoutParameters, "/TestDefaultFeedersPosition.svg", new DefaultDiagramStyleProvider());
-
-        // re-build substation graph, write SVG using the shifted feeders positioner and compare to reference (same layout providers)
-        g = graphBuilder.buildSubstationGraph(substation.getId(), false);
-
-        new HorizontalSubstationLayoutFactory().create(g, new PositionVoltageLevelLayoutFactory()).run(layoutParameters);
-        compareSvg(g, layoutParameters.setShiftFeedersPosition(true), "/TestShiftFeedersPosition.svg", new DefaultDiagramStyleProvider());
+        compareJson(g, "/TestDefaultFeedersPosition.json");
```

Comment:
Comment for all model tests: Sonar complains about this because there is no JUnit assertion. It is because Sonar is not smart enough to see that compareJson internally perform assertions. I propose in order to make Sonar happy to replace by something like:
```java
asserEquals(toJson(g), toString("/TestDefaultFeedersPosition.json"));
```

Reply:
Done